lightning-rapid-gossip-sync/src/processing.rs

```diff
@@ -194,7 +194,13 @@ impl<NG: Deref<Target=NetworkGraph<L>>, L: Deref> RapidGossipSync<NG, L> where L
 				synthetic_update.htlc_maximum_msat = htlc_maximum_msat;
 			}
 
-			network_graph.update_channel_unsigned(&synthetic_update)?;
+			match network_graph.update_channel_unsigned(&synthetic_update) {
+				Ok(_) => {},
+				Err(LightningError { action: ErrorAction::IgnoreDuplicateGossip, .. }) => {},
+				Err(LightningError { action: ErrorAction::IgnoreAndLog(_), .. }) => {},
+				Err(LightningError { action: ErrorAction::IgnoreError, .. }) => {},
```

Comment:
Note sure how exhaustive we need to be, but guessing the test only covers one of these cases?

Reply:
Yes, there are number of scenarios due to which underlying function could return those errors.
From Api perspective, we just want to ignore all the Ignore* ones. Imo, doesn't add much value in trying to replicate each one of them without a mocking-support at hand.

* Expanded the commit message, removed issue from from commit msg as well.